examples/confetti/src/main/scala/com/example/confetti/Confetti.scala

```diff
@@ -33,27 +34,36 @@ object Confetti extends IndigoDemo[Unit, Unit, Model, Unit]:
   def initialViewModel(startupData: Unit, model: Model): Outcome[Unit] =
     Outcome(())
 
-  def updateModel(context: FrameContext[Unit], model: Model): GlobalEvent => Outcome[Model] =
-    case FrameTick if context.mouse.leftMouseIsDown =>
-      Outcome(
-        model
-          .spawn(context, 15)
-          .update
-      )
+  private def maybeMouseUp(mouse: Mouse): Option[LmbOrRmb] =
+    if mouse.released(MouseButton.LeftMouseButton) then Some(MouseButton.LeftMouseButton)
+    else if mouse.released(MouseButton.RightMouseButton) then Some(MouseButton.RightMouseButton)
+    else None
```

Comment:
Feels like this could be a general function on `Mouse`? (I'm reading top to bottom so apologies if this exists further down).

Reply:
Hmmm, feels like this could be added to `Mouse` as something like `def maybeMouseReleased(mouse: Mouse): Option[MouseButton]`. The problem with that function is that it returns the "first" button released (or pressed) in the events list. Not sure how useful could that be, even the behavior could be a bit surprising for the user because the order or the `if`s inside that function would be arbitrary 🤔